samples/sample_text.cpp

```diff
@@ -56,4 +56,23 @@ int main(int /*argc*/, const char * /*argv*/ []) {
   }
 
   printf("The FlatBuffer has been parsed from JSON successfully.\n");
+
```

Comment:
The samples are meant to be simple example on how to use FlatBuffers, so this code is cluttering it a bit. Please move to `tests/test.cpp`